packages/verifiable-credentials/src/credentials.ts

```diff
@@ -128,29 +144,50 @@ export default class Credentials {
 	 * @param data 
 	 * @returns 
 	 */
-	async createCredentialJWT(data: any): Promise<object> {
+	async createCredentialJWT(subjectId: string, data: any, expirationDate?: string): Promise<object> {
+		// Ensure a credential schema has been specified
+		if (!data.schema) {
+			throw new Error('No schema specified')
+		}
+
+		// Ensure data matches specified schema
+		const schema = await this.context.getClient().getSchema(data.schema)
+		const isValid = await schema.validate(data);
+
+		// @todo: Check the schema is a "credential" schema type?
+
+		if (!isValid) {
+			throw new Error('Data does not match specified schema')
+		}
+
 		const issuer = await this.createIssuer();
 		const account = this.context.getAccount();
 		const did = await account.did();
 
-		const credential = {
+		const vcPayload: any = {
 			'@context': [
 				'https://www.w3.org/2018/credentials/v1',
 				'https://www.w3.org/2018/credentials/examples/v1',
```

Comment:
Should the example be included here?

Reply:
Yes, it was included in the @context ref in the documentation

Reply:
I don't understand this. Leaving for @nick-verida